Documentation/api-reference.md

```diff
@@ -182,6 +251,19 @@ IdentityItem item = new IdentityItem("user@example.com");
 Identity.removeIdentity(item, "Email");
 ```
 
+#### Kotlin
+
+##### Syntax
+```kotlin
+fun removeIdentity(item: IdentityItem, namespace: String)
```

Comment:
some of these params are optional without these changes https://github.com/adobe/aepsdk-edgeidentity-android/pull/97